hypothesis-python/src/hypothesis/core.py

```diff
@@ -240,6 +240,7 @@ def wrapped_test(*arguments, **kwargs):
             raise InvalidArgument(message)
 
         wrapped_test.is_hypothesis_test = True
+        wrapped_test.hypothesis = HypothesisHandle(test, wrapped_test, given_kwargs)
```

Comment:
I think this would be a lot easier to read with keyword arguments.

Reply:
should I change the main call inside given too? at the moment it's consistent with that.

Reply:
```TypeError: __init__() got an unexpected keyword argument '_get_fuzz_target'```

when I try and call with kwargs.